spec/lib/msf/core/exploit/remote/kerberos/ticket_spec.rb

```diff
@@ -150,6 +150,19 @@
       it { expect(subject.elements[0][:data]).to eq(pac_type) }
     end
   end
+
+  describe '#print_contents' do
+    subject do
+      mod = ::Msf::Exploit.new
+      mod.extend ::Msf::Exploit::Remote::Kerberos::Client
+      mod.extend ::Msf::Exploit::Remote::Kerberos::Ticket
+      mod.send(:initialize)
+      mod
+    end
+    it 'print the debug info' do
+      subject.print_contents('/Users/dwelch/.msf4/loot/20221005132420_default_192.168.176.3_mit.kerberos.cca_385969.bin')
```

Comment:
I think this is missing assertions

Reply:
I have removed this but it's still showing in the diff and I don't know why